docs/csharp/language-reference/operators/patterns.md

```diff
@@ -72,13 +72,13 @@ Beginning with C# 9.0, you can use the [negated](#pattern-combinators) `null` co
 
 ## Discard pattern
 
-Beginning with C# 8.0, you use the *discard pattern* `_` to match any expression, including `null`, as the following example shows:
+Beginning with C# 8.0, you use a *discard pattern* `_` to match any expression, including `null`, as the following example shows:
 
 :::code language="csharp" source="snippets/patterns/DiscardPattern.cs" id="BasicExample":::
 
-The discard pattern is useful in `switch` expressions. If none of a `switch` expression's patterns matches an input, the runtime throws an exception. To guarantee that a `switch` expression handles all possible values of its input as you want, provide an arm with the discard pattern. As a `switch` expression evaluates its arms in text order, the arm with the discard pattern must be last.
+A discard pattern is useful in `switch` expressions. If none of a `switch` expression's patterns matches an input, the runtime throws an exception. To guarantee that a `switch` expression handles all possible values of its input as you want, provide an arm with a discard pattern. As a `switch` expression evaluates its arms in text order, the arm with a discard pattern must be last.
```

Comment:
I think an example here using an `enum` would be very useful for readers. Because you must handle all integer input values, I find that's my most common use of a discard. 

What do you you think?

Reply:
@BillWagner I agree. I just want to add it in the separate PR then and have this merged, if it's fine with you.

Reply:
That works. I'll approve, merge, and merge in the main branch.